src/librustc_middle/mir/interpret/error.rs

```diff
@@ -334,6 +334,8 @@ pub enum UndefinedBehaviorInfo {
     Ub(String),
     /// Unreachable code was executed.
     Unreachable,
+    /// An enum discriminant was set to a value which was outside the range of valid values.
+    InvalidDiscriminant(ScalarMaybeUninit),
```

Comment:
It looks like somehow the enum variant moved around here? `InvalidDiscriminant` should be right below `InvalidChar`.